recipe/conda_build_config.yaml

```diff
@@ -341,7 +341,7 @@ harfbuzz:
 hdf4:
   - 4.2
 hdf5:
-  - 1.10.2
+  - 1.10.3
```

Comment:
We don't pin software here unless it **must** diverge from the global pinning.

You should send a PR to https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml#L344 instead so all packages will get re-pinned.

Reply:
Jeez I clicked on your hdf5 PR and commented here. Sorry. Disregard what I wrote above. This is correct!